frontend/src/views/ClientView.vue

```diff
@@ -7,22 +7,50 @@ import FloatLabel from "primevue/floatlabel";
 import InputText from "primevue/inputtext";
 import Textarea from "primevue/textarea";
 import { onMounted, ref } from "vue";
-import { useRoute } from "vue-router";
+import { useRoute, useRouter } from "vue-router";
 
-import { getClient } from "@/backendClient";
+import { createClient, deleteClient, getClient, updateClient } from "@/backendClient";
 import { ClientSalutation } from "@/global/types/appTypes";
 import type { ClientCreate, ClientUpdate } from "@/global/types/dataEditTypes";
+import { ROUTES } from "@/router";
 
 let userInfo = ref<ClientUpdate | ClientCreate>({});
 
 let birthdayDate = ref<Date>();
 
 const genders = Object.values(ClientSalutation);
 
+const router = useRouter();
 const route = useRoute();
+/**
+ * We're in editing mode if client id is present.
+ */
+const isEditing = Boolean(route.params.id);
 
+const onSaveClient = async () => {
+  /**
+   * Check what do we want to do - update or create - depending on the route.
+   */
+  if (!isEditing) {
```

Comment:
Generally, we should try to avoid negations in if-conditions. It is always easier to understand if we have a positive assertion (i.e. without the "!" in this case and switching the execution blocks.